data/domain/world-6/summoning.tsx

```diff
@@ -345,7 +354,7 @@ export class Summoning extends Domain {
                 colorBattles = summoning.summonBattles.allBattles[index];
             }
 
-            summoning.summonEssences.push({ color: index, quantity: value, display: shouldDisplay, victories: colorVictories, battles: colorBattles });
+            summoning.summonEssences.push({ color: index, quantity: value, unlocked: unlocked, display: display, victories: colorVictories, battles: colorBattles });
```

Comment:
Why do we need both `unlocked` and `display`? I'm a bit unclear from just looking at this code

Reply:
Display is used to hide last unused yet essences
Unlocked is used to know which essence should be showed in the currency list (avoid showing all and having the locked one at 0 at all times